versioned_docs/version-v1.6/upgrade/v1-5-0-to-v1-6-0.md

```diff
@@ -77,4 +77,43 @@ kubectl patch -n longhorn-system volume <volume> \
   -p '{"status":{"currentMigrationNodeID":""}}'
 ```
 
-Related issues: [#8949](https://github.com/harvester/harvester/issues/8949) and [#11479](https://github.com/longhorn/longhorn/issues/11479)
\ No newline at end of file
+Related issues: [#8949](https://github.com/harvester/harvester/issues/8949) and [#11479](https://github.com/longhorn/longhorn/issues/11479)
+
+### 3. Nodes stuck at `Waiting Reboot` due to network setup error
+
+When Harvester is installed with mgmt interface on vlan-id 1 or no vlan-id, during upgrade from v1.5.x to v1.6.0, the nodes will be stuck at `waiting reboot` due to network setup error and console logs of the node show errors as `yaml: line did not find expected key`.
```

Comment:
If the node was initially installed with Harvester v1.2.1 or earlier and continuously upgraded, it might hit following error when upgrade to v1.6.0 ...